index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <div id="main">
+      <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+      <div class="title">
+        <h1 id="title">Jack Dorsey Tribute Page</h1>
+        <div id="img-div">
+          <img
+            class="jack__img"
+            src="https://specials-images.forbesimg.com/imageserve/5ecd4871798e4c00060d2f7b/960x0.jpg?fit=scale"
+            alt="jack dorsey"
+            id="image"
```

Comment:
- [ ] Kindly phrase your comments as more of suggestions instead of orders. For example, if you need a student to do something you already mentioned you can say, " As already mentioned above, let's please remove unused id and keep our codebase clean."

Reply:
- [ ] Noted!. Thanks so much